### ISSUE TITLE
Include Bioconductor?

### ISSUE THREAD

**Issue by seandavi** [Impactstory/depsy#75]
The Bioconductor project now includes more than 1200 packages, some of which see several thousand downloads per month.  The impact on the biology community of this project is large.  The project has also been recognized by Nature as a model for software repositories. Similar in design to CRAN, it seems like it might be fairly straightforward to build support for Bioconductor into Depsy.  If you like, point us to where we might focus our efforts to help you do this.

**Comment by llrs** [comment]
I got a response via twitter: They run out of money and won't do it.
<blockquote class="twitter-tweet" data-conversation="none" data-lang="ca"><p lang="en" dir="ltr">Alas no.  Our grant has run out :(   Sorry about that, and glad you like what we built.  The code is open source in case anyone else wants to build on it!</p>&mdash; depsy (@depsy_org) <a href="https://twitter.com/depsy_org/status/968502406316109824?ref_src=twsrc%5Etfw">27 de febrer de 2018</a></blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>

But it seems that PR are wellcome, so it might be worth to write somthing like  depsy/providers/cran_reverse_dependencies.py  that works for Bioconductor.

**Comment by jasonpriem** [comment]
Yes, I'm afraid @llrs is right...this was a one-time grant from the NSF and so there is no continued funding for additional improvements (or really even for doing QA on pull requests). There are only two of us, and most of our effort is now going into Unpaywall (http://unpaywall.org...Firefox's Featured Add-On for this month, I might add! 😀) 

Unfortunately this is not super clear from the Depsy site, and so we are planning to put up a little banner that says something about this not being under development any more. Have not found time to do that yet (see above :)

One bit of good news, though, is the Bioconductor projects will be represented in our work for the followup grant to Depsy, from the Sloan foundation. This will focus more on citation from the literature, and will incorporate lots that we learned as part of the Depsy project. More detail [on the blog](http://blog.impactstory.org/collaborating-635k-grant-improve-credit-research-software/).

I wish there was a good sustainability model for projects like Depsy, but of course that is something that lots of people have already discussed at some length so will not add too much to that conversation here. Hopefully we were able to prompt some good conversations, which was the main goal of the project.
